driver/ppm_fillers.c

```diff
@@ -1261,6 +1261,7 @@ int f_proc_startupdate(struct event_filler_arguments *args)
 		exe_file = ppm_get_mm_exe_file(mm);
 
 		if (exe_file != NULL) {
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 39, 0)
```

Comment:
2.39.0 or 2.6.39 ? Also, are versions older than that supported anywhere nowadays?

Reply:
Uh i am stupid, thanks!

Reply:
Yep we still have some 2.6.32 kernels for centos on our dbg :)